pkg/ingester/ingester_v2_test.go

```diff
@@ -743,3 +744,85 @@ func newIngesterMockWithTSDBStorage(ingesterCfg Config, registerer prometheus.Re
 
 	return ingester, cleanup, nil
 }
+
+func TestIngester_V2Startup(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name  string
```

Comment:
You could remove the `name` just using a `tests := map[string]struct{}`. Resulting test cases may be more clear to read.

Reply:
Fixed